ansible/inventory/testing/group_vars/discourse/vars.yml

```diff
@@ -15,3 +15,4 @@ dis_smtp_pass: !vault |
 dis_le_email: cafe-testing@privacylx.org
 dis_developer_emails:
   - cafe-testing@privacylx.org
+dis_version: tests-passed
```

Comment:
Hum. I'm a bit skeptical about this. Right now, this is the only solution since the stable releases of discourse are broken, but it should be fine generally and the should be quite unstable because it only passed the automated tests.

Apparently the QA (quality assurance) of discourse inc. is not very good but still

This also goes against your idea of making the roles idempotent. If we run this in 3 months it will not be the same as running it now.

Thoughts?

Reply:
Disregard this comment as it will be overriden by the inventorie's variables